ddtrace/internal/agent.py

```diff
@@ -155,6 +155,9 @@ def _healthcheck():
         _conn.request("GET", "info", {}, {"content-type": "application/json"})
         resp = _conn.getresponse()
         data = resp.read()
+    except Exception:
```

Comment:
A more specific exception handler here would help readers understand what to expect from `_healthcheck`

Reply:
:+1: